cma/restricted_gaussian_sampler.py

```diff
@@ -458,21 +458,31 @@ def update(self, vectors, weights):
                 np.linalg.norm(vectors[idx[i] + 1]) for i in range(lam)
             ])
             ndx = self.dx / np.linalg.norm(self.dx)
-            for ip in range(lam):
-                if np.allclose(nlist[ip], ndx):
-                    break
-                if ip == lam - 1:
-                    raise RuntimeError("no first mirrored vector found for TPA")
+            if 11 < 3:
```

Comment:
`11 < 3` will always be `False`. Line 462-475 is unreachable code.

Reply:
Right, it's a way to incomment (old) code without losing syntax check or highlighting.